### DISCUSSION TITLE
QField app. Project download stops at 22%

### DISCUSSION THREAD

**Discussion by sboillat** [opengisch/QField#5735] [Q&A]
I have created a project on QFieldCloud (latest version of QField Cloud, QGIS 3.34.4 LTR) , with about 25 collaborators with reporter rights. Some of them (about 50%) cannot download the project with the QField app on their mobiles. Download systematically stops at 22%. Any explanation/solution for this? Thank you for your collaboration.
![Screenshot_20241015-090239](https://github.com/user-attachments/assets/93b7de70-366c-434c-ac9d-096c2f0a0ac0)

**Comment by meyerlor** [comment]
I Have now the same problem with a formerly working project - the download always stops at around 80%

**Comment by sboillat** [comment]
It's actually about 80% of the collaborators who cannot upload the project. A diversity of mobile devices including android and apple are affected.

**Reply by meyerlor** [reply]
I opened a support ticket through the qfield cloud website - i'll post updates here if there is a solution for this

**Reply by sboillat** [reply]
thank you for this

**Reply by meyerlor** [reply]
today all affected devices are working and i can download the whole project. I did not change anything

The support suggested to try and create the cloud project with the 4.9.1 version of QField Sync - i did not do that, because of the again working download, but maybe you want to try it if your problem still persists!
